components/clarinet-cli/src/devnet/package.rs

```diff
@@ -0,0 +1,128 @@
+use std::fs::{read_to_string, File};
+use std::io::{self, Write};
+use std::process;
+
+use clarinet_deployments::types::DeploymentSpecificationFile;
+use clarinet_files::NetworkManifestFile;
+
+use toml::value::Value;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Project {
+    name: String,
+    description: Option<String>,
+    authors: Vec<String>,
+    telemetry: Option<bool>,
+    cache_dir: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ClarinetSpecificationFile {
```

Comment:
This struct already exists, so you can use it instead: https://github.com/hirosystems/clarinet/blob/6b35ed75cdd45731b746b8d76c9fe3f085f308c2/components/clarinet-files/src/project_manifest.rs#L27